src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx

```diff
@@ -92,6 +95,38 @@ const Polygons = (props: IPolygonProps) => {
   } = contextMapArea;
 
   const polygonMenu = useMemo(() => props.menu || [], [props.menu]);
+
+  const filteredPolygons = useMemo(() => {
+    if (!validFilter || validFilter === "all") {
+      return polygonMenu;
+    }
+
+    return polygonMenu.filter(polygon => {
+      const status = polygon.validationStatus;
+
+      if (validFilter === "not_checked") {
+        return !status || status === "" || status === "undefined" || status === "null" || status === "notChecked";
```

Comment:
Please either use `status == null` instead of `!status` or condense this to cover both null and empty string with `isEmpty(status)` from lodash. 

When checking against several string values I sometimes use array functions instead. So another approach would be to define an array outside the component:
```
const INVALID_STATUSES = ["undefined", "null", "notChecked"];
```
and then this line becomes
```
return isEmpty(status) || INVALID_STATUSES.includes(status)
```